src/config/widgetConfig.ts

```diff
@@ -12,25 +10,3 @@ createConfig({
   },
   preloadChains: false,
 });
-
-export const widgetConfig: Partial<WidgetConfig> = {
-  tokens: {
-    allow: [
-      // {
-      //   // Wrapped SOL
-      //   chainId: ChainId.SOL,
-      //   address: 'So11111111111111111111111111111111111111112',
-      // },
-      {
-        // USDC
-        chainId: ChainId.SOL,
```

Comment:
Is that expected to remove sol?

Reply:
Yes, I tried it with that and it is different from what we have in prod - because it was overwritten anyways. We can take another look and see if we want to add **only** USDC for Solana chain tokens. The way it is right now in prod is to see all @tcheee Wdyt?